### ISSUE TITLE
Address Case Search Critical Accessibility Issues

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#841]
Critical Accessibility updates to be made to the Case Search functionality. See attached HTML file for additional information. 

Note: these changes will need to be made before we make CAMS available to USTP, as our products are contractually obligated to comply with WCAG.

- [ ] Add helper text between Filters and the first field. The helper text will say "As filters are applied, cases will appear in the search results automatically." (WCAG 3.2.2).
- [ ] Ensure each filter control has a basic accessible description that mentions that the controls will update the search results automatically. (WCAG 3.2.2).
- [ ] Ensure the keyboard focus does not disappear when a search executes. (WCAG 2.4.7).
- [ ] Ensure combo boxes can be closed without navigating away from the component. (WCAG 2.4.3).
- [ ] Case Number input should receive focus before its clear button, not after. (WCAG 2.4.3).
- [ ] Clarify tab behavior of combo boxes (ensure it doesn't jump the user around within the dropdown). (WCAG 2.4.3).
- [ ] Ensure Case Number has an input mask with helper text that says "For example, 12-34567". See USWDS for guidance (WCAG 3.2.2).
- [ ] Ensure the pieces of a combo box are appropriate tagged as combo boxes in ARIA. (WCAG 4.1.2).
- [ ] Ensure the pagination controls are links and not buttons, because they take the user to a new page (we can clarify this). (WCAG 4.1.2).
- [ ] Combo box pill buttons need some kind of screen reader description to say that you can click them to clear them from the search. (WCAG 3.2.2).
- [ ] Combo box pill buttons also need cues to inform the user that an item in the combo box is selected (via HTML or ARIA roles). (WCAG 4.1.2)
- [ ] The status message that appears when searches don't execute correctly should be given the Alert role. We also should probably put an alert on the Loading indicator, and maybe have an invisible (but voiced) alert for when a search finishes loading. (WCAG 4.1.3)
- [ ] Dropdown menu text, case number input, combo box pills, and combo box clear button do not respond to adjustments to text spacing. (WCAG 1.4.12)
- [ ] Clear control on combo boxes is too small; touch target needs to be at least 24px tall. (WCAG 2.5.8)
